### PR TITLE
TOMEE-2573-Translate to portuguese-examples-cdi-qualifier

### PR DESCRIPTION
**Pull request by Daniel-Dos**
Link to Jira : https://issues.apache.org/jira/browse/TOMEE-2573

### CODE REVIEW

File: examples/cdi-qualifier/README_pt.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
+= CDI @Qualifier
+:index-group: CDI
+:jbake-type: page
+:jbake-status: published
+
+== Introdução
+
+As vezes nós need escrever várias implementações para uma interface de regras de negócios, para aumentar o desacoplamento vamos injetar apenas a interface e o CDI vai escolher a implementação correta, para ajudar o CDI com esta escolha nós criamos os **qualificadores**.
```
Comment (marcoantoniobferreira) [inline examples/cdi-qualifier/README_pt.adoc]: in the sentence "nós need escrever", i think that you forgot to translate =)

File: examples/cdi-qualifier/README_pt.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+@Retention(RUNTIME)
+@Target({ TYPE, FIELD, METHOD })
+@Qualifier
+public @interface PaymentQualifier {
+
+	PaymentType type();	
+}
+....
+
+Este qualificador tem um método chamado `type()`, tEsse método ajudará o CDI a injetar corretamente a implementação. veja este enum:
```
Comment (marcoantoniobferreira) [inline examples/cdi-qualifier/README_pt.adoc]: in the sentence: "métódo chamado type(), 'tEsse' método". The word 'esse' don't have "t"

Comment (Daniel-Dos) [reply examples/cdi-qualifier/README_pt.adoc]: done

File: examples/cdi-qualifier/README_pt.adoc (hunk 1 above)
Comment (Daniel-Dos) [reply examples/cdi-qualifier/README_pt.adoc]: done.

**Pull request merged**